CRM/Utils/API/MatchOption.php

```diff
@@ -126,7 +125,7 @@ public function fromApiInput($apiRequest) {
           break;
 
         default:
-          // be forgiveful of sloppily api calls
+          // be forgiving of sloppy api calls
```

Comment:
In digging, it looks like I put in that line. Can't remember if it was willfully ironic or just incidental.

Let's call it willful...